ratpack-core/src/test/groovy/ratpack/http/client/HttpClientSmokeSpec.groovy

```diff
@@ -291,6 +292,27 @@ class HttpClientSmokeSpec extends HttpClientSpec {
     response.statusCode == 500
   }
 
+  def "can set connect timeout"() {
+    setup:
+    def nonRoutableIp = '192.168.0.0'
+
+    when:
+    handlers {
+      get { HttpClient httpClient ->
+        httpClient.get("http://$nonRoutableIp".toURI()) {
+          it.connectTimeout(Duration.ofMillis(20))
+        } onError {
+          render it.class.name
```

Comment:
how about just `render "error"` and assert against that just in case we need to change the underlying behavior in the future?